src/main/java/org/isf/utils/log/LogUtil.java

```diff
@@ -43,8 +43,8 @@ public static String getLogFileAbsolutePath() {
 		Logger rootLogger = Logger.getRootLogger();
 		Appender appender = rootLogger.getAppender("RollingFile");
 
-		if (appender instanceof DailyRollingFileAppender) {
-			DailyRollingFileAppender rollingFileAppender = (DailyRollingFileAppender) appender;
+		if (appender instanceof RollingFileAppender) {
+			RollingFileAppender rollingFileAppender = (RollingFileAppender) appender;
 			String relativePath = rollingFileAppender.getFile();
 			String absolutePath = new File(relativePath).getAbsolutePath();
 			return absolutePath;
```

Comment:
I don't see the use in assigning the `new File()...` to a variable and then returning the variable.   Why not just return `new File()...`?

Reply:
Just the result of many rework... done, thanks!